test/dynamo/test_higher_order_ops.py

```diff
@@ -1529,7 +1638,7 @@ def f(x):
         assert_dict_matches_regex(
             self,
             dict(counters["graph_break"]),
-            {".*HigherOrderOperator body's output must consist of tensors only": 1},
+            {".*torch.* op returned non-Tensor dict call_function": 1},
```

Comment:
This is because we relaxed the constraint that output should be tensor or list/tuple of tensors (adding dict), check my change at ```only_consist_of```. Now ```only_consist_of``` is used for checking both input and output of a HOO body function, so the error message is updated.